backend/src/main/java/ca/bc/gov/hlth/mohums/webclient/KeycloakApiService.java

```diff
@@ -22,7 +22,9 @@ public class KeycloakApiService {
     private final String groupsPath = "/groups";
     private final String identityProviderLinksPath = "/federated-identity";
     private final String userClientRoleMappingPath = "/role-mappings/clients/";
-
+    private final String serviceAccountPrefix = "service-account-";
+    private final String usernameAttribute = "username";
+    private final String serviceAccountForbiddenMessage = "Service account cannot be accessed through UMC";
```

Comment:
Not specific to your changes since you're following existing conventions but constants should typically be static and use UPPER_SNAKE_CASE naming convention.